src/app/menu/page.tsx

```diff
@@ -1,7 +1,7 @@
 import CategoryBox from "@/components/Categories/CategoryBox";
 import TempBox from "@/components/TempBox";
 
-export default function Menu() {
+export default function About() {
```

Comment:
```suggestion
export default function Menu() {
```

No need to change this

Reply:
This is still remaining

Reply:
oh, Sorry sir I didn't observe that I previously changed /menu page name also, I thought I only did for /about one and thought you were talking about that page only, so changed that one and not /menu page. I will change them as they were now.